<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifiedGACoefDialog</class>
 <widget class="QDialog" name="ModifiedGACoefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Коэффициенты</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModifiefGACoefDialog{border-color: rgb(124, 124, 124); border-width: 1px; border-radius: 10px; border-style: solid}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Размер популяции</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Вероятность скрещивания</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Вероятность мутации</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Селекция</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Скрещивание</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сила мутации</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="popSize">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="pCross">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.700000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="pMute">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.300000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="selection">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Турнирная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Пропорциональная</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="crossoverType">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Арифметическое</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BLX-alpha</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Линейное</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Одноточечное</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Двуточечное</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Равномерное</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="mutStrength">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Слабая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сильная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Средняя</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Количество потоков</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="threadAmount">
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifiedGACoefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifiedGACoefDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
